commit 2a795180372df8924e8c3e69cdbf714c6587f49c
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Dec 3 13:15:20 2018 +0000

    staging: bcm2835-camera: Unify header inclusion defines
    
    Most of the headers use ifndef FOO_H, whilst mmal-parameters.h
    used ifndef __FOO_H.
    
    Revise mmal-parameters.h to drop the underscores and make the
    headers all consistent.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/mmal-parameters.h b/drivers/staging/vc04_services/bcm2835-camera/mmal-parameters.h
index 6d21594ee384..da21ec5e320c 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/mmal-parameters.h
+++ b/drivers/staging/vc04_services/bcm2835-camera/mmal-parameters.h
@@ -19,8 +19,8 @@
  * @{
  */
 
-#ifndef __MMAL_PARAMETERS_H
-#define __MMAL_PARAMETERS_H
+#ifndef MMAL_PARAMETERS_H
+#define MMAL_PARAMETERS_H
 
 /** Common parameter ID group, used with many types of component. */
 #define MMAL_PARAMETER_GROUP_COMMON            (0 << 16)
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1729_2a795180372df8924e8c3e69cdbf714c6587f49c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1729_2a795180372df8924e8c3e69cdbf714c6587f49c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
